README.md

```diff
@@ -86,10 +85,9 @@ The following properties can be configured:
 		</tr>
 
 		<tr>
-			<td><code>align</code></td>
```

Comment:
in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.